docs/index.rst

```diff
@@ -24,13 +24,20 @@ Delta filters are added in the same way as any other filters.
 
   - Configure the parameters of the delta filter
 
-    - **Tolerance %**:  The percentage tolerance when comparing reading data. Only values that differ by more than this percentage will be considered as different from each other.
+    - **Tolerance measure**:  Tells whether *Tolerance value* is specified as a percentage or an absolute value.
```

Comment:
given other display name, May be better to keep

Tolernace Measure

Reply:
Updated